components/eventing-controller/pkg/handlers/utils.go

```diff
@@ -129,7 +129,7 @@ func getQos(qosStr string) (types.Qos, error) {
 }
 
 func getInternalView4Ev2(subscription *eventingv1alpha1.Subscription, apiRule *apigatewayv1alpha1.APIRule,
-	defaultWebhookAuth *types.WebhookAuth, defaultProtocolSettings *eventingv1alpha1.ProtocolSettings,
+	defaultWebhookAuth types.WebhookAuth, defaultProtocolSettings *eventingv1alpha1.ProtocolSettings,
```

Comment:
How about the `ProtocolSettings`? Would changing it reflect in the default config as well?

Reply:
No, we are dealing with values for exemptHandshake and contentMode later.